release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -785,6 +785,10 @@ You can now configure `linuxptp` services as a grandmaster clock (T-GM) for dual
 
 The host system clock is synchronized from the NIC that is connected to the Global Navigation Satellite Systems (GNSS) time source. The second NIC is synced to the 1PPS timing output provided by the NIC that is connected to GNSS. For more information, see xref:../networking/ptp/configuring-ptp.adoc#configuring-linuxptp-services-as-grandmaster-clock-dual-nic_configuring-ptp[Configuring linuxptp services as a grandmaster clock for dual E810 NICs].
 
+[id="ocp-4.17-masquerade-subnet-change_{context}"]
+==== Masquerade subnet change for new clusters
+For OpenShift Container Platform 4.17 and later versions, clusters use 169.254.0.0/17 for IPv4 and fd69::/112 for IPv6 as the default masquerade subnet. These ranges should be avoided by users. For upgraded clusters, there is no change to the default masquerade subnet.
```

Comment:
```suggestion
For {product-title} {product-version} and later versions, clusters use `169.254.0.0/17` for IPv4 and `fd69::/112` for IPv6 as the default masquerade subnet. These ranges should be avoided by users. For upgraded clusters, there is no change to the default masquerade subnet.
```

Should this note suggest that users change these values? What if they do decide to use them, what will happen? Is this a known issue? Upgraded clusters use different default values that can be used without issues?

Reply:
A CM might be needed for this release note as it's something a customer that creates a new cluster, really needs to know. Stakeholders could inform them directly. I'm assuming all platforms are impacted.

Reply:
@dfitzmau this note matches those in the docs for 4.17. This is a 4.17 only issue so I'm hardcoding the version. We are treating this as a bug that doesn't need CM because it is already known and reported in the main docs. With respect to your other questions, no this is for new clusters only and the note specifies that these values/ranges should be avoided.